src/places.js

```diff
@@ -24,6 +25,20 @@ function createPlacesClient(algoliasearch) {
     var client = algoliasearch(appID, apiKey, opts);
     var index = client.initIndex('places');
     index.search = buildSearchMethod('query', '/1/places/query');
+    index.reverse = function(aroundLatLng, options, callback) {
+      var encodedOptions = qs3.encode(options);
+      var encoded = ['aroundLatLng=' + encodeURIComponent(aroundLatLng)]
```

Comment:
why is `aroundLatLng` not one of the options instead? This code would be simpler like that

Reply:
I wanted to have an interface that was similar to the one we have for search e.g. `search(query, options, cb)`, with `aroundLatLng` being the "query" part for a reverse geocoding query but I agree that it would indeed be much simpler to have everything in the options field instead.

Will update 👍

Reply:
I changed the method interface to support both `reverse(aroundLatLng: string)` and `reverse(options: object)`.